src/nvim/vim.h

```diff
@@ -287,9 +287,11 @@ enum { FOLD_TEXT_LEN = 51 };  //!< buffer size for get_foldtext()
 // functions of these names. The declarations would break if the defines had
 // been seen at that stage.  But it must be before globals.h, where error_ga
 // is declared.
-#define mch_errmsg(str)        fprintf(stderr, "%s", (str))
+#ifndef WIN32
+# define mch_errmsg(str)        fprintf(stderr, "%s", (str))
+# define mch_msg(str)           printf("%s", (str))
+#endif
 #define display_errors()       fflush(stderr)
```

Comment:
`display_errors` could be removed completely. It is only used in `do_shell()`, left over from Vim. `do_shell()` in Nvim uses the Nvim UI, not the tty directly.